docs/generated/manifests/menus.json

```diff
@@ -123,15 +123,15 @@
         "isExternal": false,
         "children": [
           {
-            "name": "TypeScript Packages",
+            "name": "TypeScript Monorepo",
             "path": "/getting-started/tutorials/typescript-packages-tutorial",
             "id": "typescript-packages-tutorial",
             "isExternal": false,
             "children": [],
             "disableCollapsible": false
           },
           {
-            "name": "React Monorepo",
+            "name": "React-Focused TypeScript Monorepo",
```

Comment:
Maybe it's okay to keep `React Monorepo` for now so it's consistent with the rest.